src/libstd/sys/windows/mod.rs

```diff
@@ -42,7 +42,27 @@ pub mod thread_local;
 pub mod time;
 pub mod stdio;
 
-pub fn init() {}
+pub fn init() {
+    use alloc::oom;
+    // A nicer handler for out-of-memory situations than the default one. This
+    // one prints a message to stderr before aborting.
```

Comment:
In the spirit of deduplicating information, it should be fine to leave a comment here which just points to the unix impl explaining what's going on.